index.js

```diff
@@ -21,7 +21,11 @@ module.exports = function load (src, opts, cb) {
   }
 
   if (opts.text) {
-    script.appendChild(document.createTextNode(opts.text))
+    if (window.addEventListener) {
+      script.appendChild(document.createTextNode('' + opts.text))
+    } else {
+      script.text = '' + opts.text
```

Comment:
works for me.  Is there a way I can run these tests locally without you guys having to merge the changes?